contracts/polygon/connectors/aave/v3-import/main.sol

```diff
@@ -16,11 +16,16 @@ contract AaveV3ImportResolver is AaveHelpers {
 		internal
 		returns (string memory _eventName, bytes memory _eventParam)
 	{
-		require(
-			AccountInterface(address(this)).isAuth(userAccount),
-			"user-account-not-auth"
-		);
-
+		if (
+			ListInterface(0x839c2D3aDe63DF5b0b8F3E57D5e145057Ab41556).accountID(
+				userAccount
+			) == 0
```

Comment:
@Richa-iitr we already have a hardcoded constant variable,right?

Reply:
Yes, updating.